lib/srv/forward/sshserver.go

```diff
@@ -851,6 +850,84 @@ func (s *Server) handleConnection(ctx context.Context, chans <-chan ssh.NewChann
 	}
 }
 
+// handleClientChannels handles channel open requests from the remote server.
+func (s *Server) handleClientChannels(ctx context.Context) {
+	forwardedTCPIP := s.remoteClient.HandleChannelOpen(teleport.ChanForwardedTCPIP)
```

Comment:
Is it possible that we end up accepting the forwarding before this goroutine gets here and sets up the channel open?